EFCore.NamingConventions/NamingConventions/Internal/NamingConventionsOptionsExtension.cs

```diff
@@ -11,17 +11,23 @@ public class NamingConventionsOptionsExtension : IDbContextOptionsExtension
     {
         DbContextOptionsExtensionInfo _info;
         NamingConvention _namingConvention;
+        CultureInfo _cultureInfo;
 
-        public NamingConventionsOptionsExtension() {}
 
-        protected NamingConventionsOptionsExtension([NotNull] NamingConventionsOptionsExtension copyFrom)
-            => _namingConvention = copyFrom._namingConvention;
+        public NamingConventionsOptionsExtension(CultureInfo info = null) {}
+
+        protected NamingConventionsOptionsExtension([NotNull] NamingConventionsOptionsExtension copyFrom, CultureInfo cultureInfo = null)
+        {
+            _namingConvention = copyFrom._namingConvention;
+            _cultureInfo = cultureInfo;
+        }
 
         public virtual DbContextOptionsExtensionInfo Info => _info ??= new ExtensionInfo(this);
 
-        protected virtual NamingConventionsOptionsExtension Clone() => new NamingConventionsOptionsExtension(this);
+        protected virtual NamingConventionsOptionsExtension Clone(CultureInfo cultureInfo = null) => new NamingConventionsOptionsExtension(this, cultureInfo);
```

Comment:
Remove this parameter. Clone accepts no parameter - it's job is to return a new instance with the same settings as the current instance. So it's responsible for copying across the naming convention, and needs to be updated to copy the new culture as well.